Quality/MK8_1080p/rules.txt

```diff
@@ -18,6 +18,22 @@ formatsExcluded = 0x41A # exclude obvious textures
 overwriteWidth = 960
 overwriteHeight = 540
 
+# Multiplayer-Mode Resolution (2p)
+[TextureRedefine]
+width = 640
+height = 720
+formatsExcluded = 0x41A # exclude obvious textures
+overwriteWidth = 960
+overwriteHeight = 1080
+
+# Multiplayer-Mode Resolution (4p)
+[TextureRedefine]
+width = 640
+height = 360
+formatsExcluded = 0x41A # exclude obvious textures
+overwriteWidth = 960
+overwriteHeight = 1080
```

Comment:
Your current screenshots are supersampling vertical due to the incorrect height setting I pointed out. You can see it on the 1440p screenshots.

I'm not sure how overwriting rules work in different files, it might help if exzap added some kind of order number to assign to graphics packs so that certain ones could overwrite others on demand. (eg. a multiplayer texture rule would overwrite full-res alpha one in the normal texture pack when enabled)